tableau_utilities/hyper/hyper.py

```diff
@@ -0,0 +1,112 @@
+import os
+import shutil
+from zipfile import ZipFile
+from tableauhyperapi import HyperProcess, Connection, Telemetry, CreateMode, TableDefinition, TableName, SqlType
+
+from tableau_utilities.tableau_file.tableau_file import TableauFileError
```

Comment:
```suggestion
from tableau_utilities.tableau_file.tableau_file import TableauFileError, Datasource
```
Should include `Datasource` for the type suggestion on the function arguments.